### PR TITLE
Update README.md to be consistent across all drivers

### PR DESCRIPTION
**Pull request by Dominitio**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [ ] Local run of `mvn install` succeeds
- [ ] My code follows the code style of this project
- [ ] My change requires a change to the Javadoc documentation
- [ ] I have updated the Javadoc documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License

- [ ] I confirm that this pull request can be released under the Apache 2 license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
 
 Version 1.x is still supported.
 
+For getting started with the driver, see [Java and Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.java.html)
```
[the remark this answers is not in the document]
Comment (Dominitio) [reply README.md:14]: Done

File: README.md [hunk 2]
```diff
@@ -21,6 +31,52 @@ information about Amazon QLDB Driver for Java requirements, see [Amazon QLDB Dri
 1. **Using the Amazon QLDB Driver for Java** &mdash; The best way to get familiar with the Amazon QLDB Driver for Java 
 is to read [Getting Started with the Amazon QLDB Driver](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started-driver.html) in the [Amazon QLDB Developer Guide](https://docs.aws.amazon.com/qldb/latest/developerguide/what-is.html).
 
+### See also
+
+1. [Getting Started with Amazon QLDB Java Driver](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.java.html) A guide that gets you started with executing transactions with the QLDB Java driver.
+2. [QLDB Java Driver Cookbook](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-cookbook-java.html) The cookbook provides code samples for some simple QLDB Java driver use cases. 
+3. [Amazon QLDB Java Driver Tutorial](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.java.tutorial.html): In this tutorial, you use the QLDB Driver for Java to create an Amazon QLDB ledger and populate it with tables and sample data.
```
[the remark this answers is not in the document]
Comment (Dominitio) [reply README.md:38]: I say, they may refer to the same page, but the their focus and contexts are different.

File: README.md [hunk 3]
```diff
@@ -21,6 +31,52 @@ information about Amazon QLDB Driver for Java requirements, see [Amazon QLDB Dri
 1. **Using the Amazon QLDB Driver for Java** &mdash; The best way to get familiar with the Amazon QLDB Driver for Java 
 is to read [Getting Started with the Amazon QLDB Driver](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started-driver.html) in the [Amazon QLDB Developer Guide](https://docs.aws.amazon.com/qldb/latest/developerguide/what-is.html).
```
[the remark this answers is not in the document]
Comment (Dominitio) [reply README.md:32]: It seems fine to me.

File: README.md [hunk 4]
```diff
@@ -21,6 +31,52 @@ information about Amazon QLDB Driver for Java requirements, see [Amazon QLDB Dri
...
+format offering interchangeable binary and text representations. For more information read the [Ion docs](http://amzn.GitHub.io/ion-docs/) for [Amazon Ion Java](https://github.com/amzn/ion-java).
+2. **Using PartiQL** &mdash; Amazon QLDB uses PartiQL to send requests to the ledger database. You can get started with 
+the [PartiQL Tutorial](https://PartiQL.org/tutorial.html). Also read the [PartiQL Reference](https://docs.aws.amazon.com/en_pv/qldb/latest/developerguide/ql-reference.html) from the Amazon QLDB Developer Guide.
+1. Refer the section [Common Errors while using the Amazon QLDB Drivers](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-errors.html) which describes runtime errors that can be thrown by the Amazon QLDB Driver when calling the qldb-session APIs.
+
+
+
+## Development
+
+### Setup
```
[the remark this answers is not in the document]
Comment (Dominitio) [reply README.md:56]: Changed

File: README.md [hunk 5]
```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
```
Comment (kylepbit) [inline README.md:11]: L3 should have a link for "Amazon Quantum Ledger Database (QLDB)" as Python has.

Comment (kylepbit) [reply README.md:11]: Additionally, the badges should be right after the header on L2, and then the intro blurb should be after.

File: README.md [hunk 6]
```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
 
 Version 1.x is still supported.
 
+For getting started with the driver, see [Java and Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.java.html).
+
+## Requirements
+
+### Basic Configuration
```
Comment (kylepbit) [inline README.md:18]: This is missing minimum Java versions.

File: README.md [hunk 7]
```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
...
+
+See [Accessing Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/accessing.html) for information on connecting to AWS.
+
 ## Getting Started
 
+Please see the [Quickstart guide for the Amazon QLDB Driver for Java](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-quickstart-java.html).
```
Comment (kylepbit) [inline README.md:24]: This is missing the list tables snippet.

File: README.md [hunk 8]
```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
...
+
 1. **Sign up for AWS** &mdash; Before you begin, you need an AWS account. For more information about creating an AWS 
 account and retrieving your AWS credentials, see [Accessing Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/accessing.html) in the Amazon QLDB Developer Guide.
 1. **Minimum requirements** &mdash; To use the Amazon QLDB Driver for Java, you'll need **Java 1.8+**. For more
```
Comment (kylepbit) [inline README.md:28]: Hmm - here's the min Java version, seems like it should be under requirements.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kylepbit) [reply README.md:32]: This should point to the java driver - https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.java.html

[the remark this answers is not in the document]
Comment (kylepbit) [reply README.md:32]: Looking further, I'm unsure this section of 4 items is really necessary, the java portion should move to min requirements, the last item is duplicated below, signing up seems self-evident, and the install is probably below as well?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kylepbit) [reply README.md:38]: The ordering here is different from other drivers, can they be aligned? For example , Driver Recommendations is ordered 2nd last in Python and others, but here is ordered in the middle.

File: README.md [hunk 9]
```diff
@@ -39,19 +95,6 @@ The GitHub issues are intended for bug reports and feature requests. For help an
 * Provides an easy-to-use programming model for working with transactions with Amazon QLDB.
```
Comment (kylepbit) [inline README.md:95]: Seems like the Features section should be removed.

File: README.md (hunk 5 above)
Comment (HomayounA) [reply README.md:11]: Changed

File: README.md (hunk 6 above)
Comment (HomayounA) [reply README.md:18]: Added java version here

File: README.md (hunk 8 above)
Comment (HomayounA) [reply README.md:28]: Moved to requirements

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (HomayounA) [reply README.md:32]: Link is updated. I think the remaining three items help to give some direction. If we remove, we should replace with a example statements (commands) to write a simple app. thoughts?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (HomayounA) [reply README.md:38]: Reordered and added (Ion Cookbook, and Common errors items)

File: README.md (hunk 9 above)
Comment (HomayounA) [reply README.md:95]: Removed

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kylepbit) [reply README.md]: I'm fine with leaving as is.

**Comment by guyilin-amazon** [comment]
Updated according to https://github.com/awslabs/amazon-qldb-driver-python

**Pull request merged**